doc/nrf/releases/release-notes-latest.rst

```diff
@@ -233,13 +247,20 @@ The following list summarizes the most important changes inherited from upstream
       * :option:`CONFIG_OPENTHREAD_SRP_CLIENT`
       * :option:`CONFIG_OPENTHREAD_SRP_SERVER`
 
+    * Fixed a bug where MAC layer code asserted after OpenThread diagnostic was stopped.
+    * Added a microsecond timer API implementation to the Zephyr port.
+
   * MQTT:
 
     * Fixed logging of UTF-8 strings.
 
   * Sockets:
 
     * Fixed TLS sockets access from user space.
+    * :option:`CONFIG_NET_SOCKETS_POSIX_NAMES` is now enabled by default.
+    * Added :c:macro:`TLS_DTLS_HANDSHAKE_TIMEOUT_MIN` and :c:macro:`TLS_DTLS_HANDSHAKE_TIMEOUT_MAX` TLS socket options.
+    * Fixed :c:func:`poll` behavior during handshake with DTLS socket.
```

Comment:
```suggestion
    * Fixed :c:func:`zsock_poll` behavior during handshake with DTLS socket.
```
Correct?
If I understand it correctly, this function can also be exposed as poll(), but it's not possible to link to that name in the docs.

Reply:
yes, `zsock_poll` is also fine if we can't use `poll`